src/View/ViewableData.php

```diff
@@ -588,9 +598,11 @@ public function obj($fieldName, $arguments = [], $cache = false, $cacheName = nu
      * @param array $arguments
      * @param string $identifier an optional custom cache identifier
      * @return Object|DBField
+     * @deprecated 5.4.0 use obj() instead
      */
     public function cachedCall($fieldName, $arguments = [], $identifier = null)
```

Comment:
Nothing uses this, so no need to replace it. If we do need this in the future it can be added to the new `CastingService`.